.github/workflows/deploy-pr.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
```

Comment:
I wonder if deploying automatically is a good idea. It could be unsafe to deploy PRs opened by external contributors, so deploying manually might be better.

Reply:
yes, we could use `workflow_dispatch` instead to be on the safe side but we will need to run it manually each time the PR is updated.

Reply:
Running manually seems fine to me.

Reply:
I think we might be able to trigger the action only on local PRs (so from branches on that repository) instead of PRs coming from forked repositories. How does that sound?